vkcs/kubernetes/resource_addon_test.go

```diff
@@ -95,6 +103,10 @@ func testAccKubernetesAddonImportStateID(s *terraform.State) (string, error) {
 	return "", fmt.Errorf("addon not found")
 }
 
+func uniqueKubernetesNodeGroupName() string {
```

Comment:
You should think of a way to make it common across all tests, it looks a bit messy with different approaches to achieve the same goal in different pieces of tests.

Reply:
Our resources have different naming requirements, for example different len, allowed characters. I prefer add testName and time to name, but this is impossible for cluster and node group

Reply:
Ok, let's leave it for now.